src/views/PoEMap.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <b-container fluid>
+    <b-row>
+      <b-col>
+        <l-map
+            ref="p-map"
+            :zoom="zoom"
+            :center="center"
+            :options="mapOptions"
+            :crs="crs"
+            :style="mapStyle"
+          >
+          <l-image-overlay :url="url" :bounds="bounds"/>
+          <l-control-layers
+            position="topright"
+            :collapsed="false"
+            :sort-layers="true"
+          />
+          <l-layer-group
+          v-for="geojson in geo"
+            layer-type="overlay"
+            :key="geojson.name"
+            :name="geojson.name"
+          >
+            <l-geo-json :geojson="geojson.json" :options="geojson.opts"/>
+          </l-layer-group>
+        </l-map>
+      </b-col>
+    </b-row>
+  </b-container>
+</template>
+
+<script>
+  import Vue from 'vue';
+  import {L} from 'vue2-leaflet';
+
+  /* map stuff */
+  import plains from '@/assets/img/plains.png';
+  import labels from '@/assets/json/geo/plains/labels.json';
+  import fish from '@/assets/json/geo/plains/fishing.json';
+  import grineer from '@/assets/json/geo/plains/grineer.json';
+  import lorefish from '@/assets/json/geo/plains/lorefish.json';
+  import wisp from '@/assets/json/geo/plains/wisp.json';
+  import lure from '@/assets/json/geo/plains/lure.json';
+  import cave from '@/assets/json/geo/plains/cave.json';
+  import grineerIcon from '@/assets/img/map_icons/grineer.png';
+  import oddityIcon from '@/assets/img/map_icons/lorefish.png';
+  import wispIcon from '@/assets/img/map_icons/wisp.png';
+  import lureIcon from '@/assets/img/map_icons/lure.png';
+  import caveIcon from '@/assets/img/map_icons/normal-cave.png';
+  import MapPopup from '@/components/MapPopup.vue';
+  import OddityPopup from '@/components/OddityPopup.vue';
+
+  const grineerMarker = L.icon({
+    iconUrl: grineerIcon,
+    iconSize: [25, 25],
+  });
+
+  const oddityMarker = L.icon({
+    iconUrl: oddityIcon,
+    iconSize: [33, 25],
+  });
+
+  const wispMarker = L.icon({
+    iconUrl: wispIcon,
+    iconSize: [50, 33],
+  });
+
+  const lureMarker = L.icon({
+    iconUrl: lureIcon,
+    iconSize: [50, 50],
+  });
+
+  const caveMarker = L.icon({
+    iconUrl: caveIcon,
+    iconSize: [25, 25],
+  });
+
+  function onEachFeature (feature, layer) {
+    let Popup = Vue.extend(MapPopup);
+    let popup = new Popup({
```

Comment:
Having two variables that differ only by casing may lead to confusion.

Reply:
it's a generic method for all popups, and the standard in js is that `Popup` is a class whereas `popup` is an instance... it could, for sure, but given it's limited usage here and unavailability outside of this function, I'm ok with it as-is